controller/customer/index.js

```diff
@@ -4,4 +4,129 @@
  * This software is the confidential information of F2FINTECH., and is licensed as
  * restricted rights software. The use, reproduction, or disclosure of this software is subject to
  * restrictions set forth in your license agreement with F2 FINTECH.
- */
\ No newline at end of file
+ */
+
+const passport = require("passport");
+const Utility = require("../../utility");
+const CustomerModel = require("../../model/customer");
+
+const loginPassport = (req, res, next) => {
```

Comment:
rename it to just login as we have similar kind of names throughout the file eg: register